processes/workflow.rst

```diff
@@ -156,12 +157,57 @@ In that case, change the ticket's status to **Won't Fix.**
 Name the duplicate ticket in the status change comment field.
 
+.. _github-orgs:
+
+DM GitHub Organizations
+=======================
+
+DM's Git repositories are available from three GitHub organizations: `lsst <https://github.com/lsst>`__, `lsst-dm <https://github.com/lsst-dm>`__, and `lsst-sqre <https://github.com/lsst-sqre>`__.
+LSST DM source code is publicly available and open source.
+
+You should already be a member of the `lsst <https://github.com/lsst>`__ and `lsst-dm <https://github.com/lsst-dm>`__ GitHub organizations.
+If you cannot create repositories or push to repositories there, ask you T/CAM to :ref:`add you to these organizations <getting-started-github>`.
+
+lsst GitHub organization
+------------------------
+
+The `lsst <https://github.com/lsst>`__ GitHub organization is for public-facing code and documentation repositories.
+Specifically, packages in main EUPS distributions are available from the `lsst <https://github.com/lsst>`__ organization, along with official documents (including LDM design documentation).
+
+lsst-dm GitHub organization
+---------------------------
+
+The `lsst-dm <https://github.com/lsst-dm>`__ GitHub organization is for miscellaneous Data Management projects:
+
+- EUPS packages that are not yet part of the official distribution. Projects can be incubated in `lsst-dm <https://github.com/lsst-dm>`__ and later migrated to the `lsst <https://github.com/lsst>`__ organization.
+- Retired projects and EUPS packages (these have names prefixed with "legacy").
+- Prototypes, internal experiments, and other types of ad-hoc projects.
+- Internal documentation, including DMTN technotes and this DM Developer Guide.
+
+lsst-sqre GitHub organization
+-----------------------------
+
+The `lsst-sqre <https://github.com/lsst-sqre>`__ GitHub organization is used by the SQuaRE team for operational services and internal experiments.
+SQuaRE's technical notes (SQR) are also available in `lsst-sqre <https://github.com/lsst-sqre>`__.
+
+Upstream repositories and organizations
+---------------------------------------
+
+Whenever possible, DM developers should contribute to the third-party open source codebases used by the LSST Stack.
+Since this type of development is typically done with a fork-and-PR workflow, the third-party repo should be forked into an LSST organization, usually `lsst-dm <https://github.com/lsst>`__ or `lsst-sqre <https://github.com/lsst-sqre>`__.
+Doing upstream development in an LSST GitHub organization lets the team more easily identify what what work is being done.
+
+Personal GitHub repositories
+----------------------------
+
+Work funded by LSST, and delivered to LSST in ticketed work, **cannot** be developed in personal GitHub repositories outside of the `lsst <https://github.com/lsst>`__, `lsst-dm <https://github.com/lsst-dm>`__, and `lsst-sqre <https://github.com/lsst-sqre>`__ GitHub organizations.
```

Comment:
Might want to say that personal repositories can be used for work by non-DM-staff contributing to DM code or for DM staff working on personal projects (including "science time").